tests/test_hvac_csc.py

```diff
@@ -0,0 +1,567 @@
+                    "vea01P01",
+                    "vea01P05",
+                    "vea08P05",
+                    "vea09P05",
+                    "vea10P05",
+                    "vea11P05",
+                    "vea12P05",
+                    "vea13P05",
+                    "vea14P05",
+                    "vea15P05",
+                    "vea16P05",
+                    "vea17P05",
+                    "vec01P01",
+                    "vex03P04",
+                    "vex04P04",
+                    "vin01P01",
+                ),
+            )
+
+    async def test_bin_script(self):
+        await self.check_bin_script(name="HVAC", index=None, exe_name="run_hvac.py")
+
+    async def test_do_chiller01P01(self):
+        async with self.make_csc(initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1):
+            await salobj.set_summary_state(remote=self.remote, state=salobj.State.ENABLED)
+
+            tel_chiller01P01 = await self.assert_next_sample(topic=self.remote.tel_chiller01P01, flush=True,)
+            self.assertTrue(tel_chiller01P01.compresor01Funcionando is False)
+            self.assertTrue(tel_chiller01P01.compresor02Funcionando is False)
+            self.assertTrue(tel_chiller01P01.compresor03Funcionando is False)
+            self.assertTrue(tel_chiller01P01.compresor04Funcionando is False)
+            self.assertTrue(tel_chiller01P01.comandoEncendido is False)
+            self.assertAlmostEqual(-99.99, tel_chiller01P01.setpointActivo, places=5)
+
+            setpoint_activo = 90
+            comando_encendido = True
+            await self.remote.cmd_chiller01P01.set_start(
+                setpointActivo=setpoint_activo, comandoEncendido=comando_encendido, timeout=STD_TIMEOUT,
+            )
+            tel_chiller01P01 = await self.assert_next_sample(topic=self.remote.tel_chiller01P01, flush=True,)
+            tel_chiller01P01 = await self.assert_next_sample(topic=self.remote.tel_chiller01P01, flush=True,)
+            self.assertTrue(tel_chiller01P01.compresor01Funcionando == comando_encendido)
+            self.assertTrue(tel_chiller01P01.compresor02Funcionando == comando_encendido)
+            self.assertTrue(tel_chiller01P01.compresor03Funcionando == comando_encendido)
+            self.assertTrue(tel_chiller01P01.compresor04Funcionando == comando_encendido)
+            self.assertTrue(tel_chiller01P01.comandoEncendido == comando_encendido)
+            self.assertAlmostEqual(setpoint_activo, tel_chiller01P01.setpointActivo)
+
+    async def test_do_crack01P02(self):
+        async with self.make_csc(initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1):
+            await salobj.set_summary_state(remote=self.remote, state=salobj.State.ENABLED)
+
+            tel_crack01P02 = await self.assert_next_sample(topic=self.remote.tel_crack01P02, flush=True,)
+            self.assertTrue(tel_crack01P02.estadoFuncionamiento is False)
```

Comment:
If you are willing to give up the "is" test and settle for equality then I think `self.assertFalse(tel_crack01P02.estadoFuncionamiento)` would be much less confusing and assert_next_sample is even better:
```
assert_next_sample(... tel_crack01P02=False, ...)
```

I admit that testing for `is` is more rigorous. If you want that then please use `assertIs`:
```
self.assertIs(tel_crack01P02.estadoFuncionamiento, False)`
```

Reply:
I opted for using assert_next_sample.